app/src/main/java/com/example/livestreamingclient/ReceiverActivity.kt

```diff
@@ -1,10 +1,69 @@
 package com.example.livestreamingclient
 
+import android.content.pm.ActivityInfo
 import android.os.Bundle
+import android.view.View
+import android.widget.ImageView
 import androidx.appcompat.app.AppCompatActivity
+import com.shuyu.gsyvideoplayer.GSYBaseActivityDetail
+import com.shuyu.gsyvideoplayer.GSYVideoManager
+import com.shuyu.gsyvideoplayer.utils.OrientationUtils
+import com.shuyu.gsyvideoplayer.video.StandardGSYVideoPlayer
+import kotlinx.android.synthetic.main.activity_receiver.*
 
 class ReceiverActivity : AppCompatActivity() {
+    lateinit var videoPlayer: StandardGSYVideoPlayer
+    lateinit var orientationUtils: OrientationUtils
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_receiver)
+
+        videoPlayer = rec_surface
+        var source1 = "http://9890.vod.myqcloud.com/9890_4e292f9a3dd011e6b4078980237cc3d3.f20.mp4"
+        videoPlayer.setUp(source1, true, "测试视频")
```

Comment:
Instead of setting here, use button and texteditor to load video source dynamically.